.github/CODEOWNERS

```
@@ -2,7 +2,9 @@
 * @LavaMoat/devs
 
 *eslint* @boneskull
-tsconfig*.json @boneskull
+tsconfig*.json @boneskull @legobeat
+yarn.lock @legobeat
+{,packages/*/}package.json @legobeat
```

```suggestion
packages/*/package.json @legobeat
package.json @legobeat
```
Was being a bit too fancy with the glob pattern there *^^